libs/core/src/lib/tabs/tab-list.component.ts

```diff
@@ -116,8 +117,8 @@ export class TabListComponent implements AfterViewInit, OnChanges, OnDestroy {
     }
 
     /** @hidden */
-    tabHeaderClickHandler(tabIndex: number): void {
-        if (this.selectedIndex !== tabIndex) {
+    tabHeaderClickHandler(tabIndex: number, disabled?: boolean): void {
```

Comment:
Wouldn't it be better to be consistent here and use `this._canBeSelected(tabIndex)` as well?

Reply:
I think we can't have example of disabled in docs, because it's not designed by fiori3